ais_service_discovery/call_service.py

```diff
@@ -6,10 +7,18 @@
 from .sqs import SqsAdaptor, Send
 from json import dumps
 
-function = client('lambda')
-sns = client('sns')
-sqs = client('sqs')
-service_discovery = client('servicediscovery')
+from botocore.config import Config
+
+
+BOTO_MAX_ATTEMPTS = int(env.get('BOTO_MAX_ATTEMPTS', 10))
+BOTO_READ_TIMEOUT = float(env.get('BOTO_READ_TIMEOUT', 300))
+
+config = Config(read_timeout=BOTO_READ_TIMEOUT, retries={'max_attempts': BOTO_MAX_ATTEMPTS})
```

Comment:
Reading [this](https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html), Should we use  `total_max_attempts` instead of `max_atempts`?

Reply:
Which also uses AWS standardised env `AWS_MAX_ATTEMPTS`

Reply:
Yep, better still, good find!

Reply:
I'll leave our env var as well just so we have two the same, as there doesn't appear to be one for `read_timeout` :(